pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprfocus.nix

```diff
@@ -3,12 +3,13 @@
   mkHyprlandPlugin,
   hyprland,
   fetchFromGitHub,
+  fetchpatch2,
   nix-update-script,
 }:
 
 mkHyprlandPlugin hyprland {
   pluginName = "hyprfocus";
-  version = "0-unstable-2024-10-09";
+  version = "0-unstable-2024-11-09";
```

Comment:
I decided to run `nix-update` on this to check if there is a new version (there wasn't) and it found this mistake